src/core/public/chrome/chrome_service.tsx

```diff
@@ -670,4 +671,10 @@ export interface InternalChromeStart extends ChromeStart {
    * @internal
    */
   getHeaderComponent(): JSX.Element;
+
+  /**
+   * Whether to use the updated header UI
```

Comment:
what does "updated" mean? Are you referring to new UI? can we use "newHeader" to align with the naming convention across the codebase